BrainPortal/app/models/cluster_task.rb

```diff
@@ -2337,22 +2337,25 @@ def singularity_commands(command_script)
     # Numbers in (paren) correspond to the comment
     # block in the script, well below.
 
-    # (1) The root of the shared area for all CBRAIN tasks
+    # (1) additional singularity execution command options defined in ToolConfig
+    container_params = self.tool_config.container_params
```

Comment:
use `.presence` to nillify the string if it's blank; unlike the other singularity options, this one is not always present